src/backend/gl/src/conv.rs

```diff
@@ -216,13 +216,54 @@ pub fn describe_format(format: Format) -> Option<FormatDescription> {
             FormatDescription::new(glow::RGBA32I, glow::RGBA_INTEGER, glow::INT, 4, Integer)
         }
         Rgba32Sfloat => FormatDescription::new(glow::RGBA32F, glow::RGBA, glow::FLOAT, 4, Float),
+        S8Uint => FormatDescription::new(glow::R8, glow::RED, glow::UNSIGNED_BYTE, 1, Float),
+        D16Unorm => FormatDescription::new(
+            glow::DEPTH_COMPONENT16,
+            glow::DEPTH,
+            glow::UNSIGNED_NORMALIZED,
+            1,
+            Float,
+        ),
+        D16UnormS8Uint => FormatDescription::new(
```

Comment:
GL doesn't have this format, no need to handle it here

Reply:
Are we sure that Vulkan Spec does not require the existence of this format for a vulkan implementation to be conformant?

Reply:
Yes, I'm sure. Vulkan guarantees for formats are super lax.

Reply:
Ok, removed.